### PR TITLE
Moved Climb-Up/Down plugin into MenuEntrySwapper

### PR DESCRIPTION
**Pull request by SRLJustin**
resolves todo: "move climbplugin functionality to menu entry swapper"

### CODE REVIEW

**Comment by Lucwousin** [comment]
I think it would be better if you just change climb with climb up/climb down. There's not really a reason to remove everything but those imo

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java [hunk 1]
```diff
@@ -128,9 +128,14 @@
 	@Getter
 	private boolean configuringShiftClick = false;
 
+	@Getter
```
Comment (Lucwousin) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java]: I don't think the getters for this are needed, but looks good other than that

**Comment by Lucwousin** [comment]
Sorry for being stupid, but while watching Null's stream he had a problem walking up stairs, caused by shift to walk. You may think how does that make you stupid? It would still work if you strip all entries except for climb up/down. Aka strippin' time!

I'll do it myself after this rune dragon task, if you don't stop me in discord :P

**Comment by Lucwousin** [comment]
oh LOL

**Comment by zeruth** [comment]
whoops, feel free to do that still, posted comment right after I merged :p

**Pull request merged**